pkg/kubectl/describe.go

```diff
@@ -382,13 +382,13 @@ func (d *ResourceQuotaDescriber) Describe(namespace, name string, describerSetti
 func helpTextForResourceQuotaScope(scope api.ResourceQuotaScope) string {
 	switch scope {
 	case api.ResourceQuotaScopeTerminating:
-		return "Matches all pods that have an active deadline."
+		return "Matches all pods that have an active deadline. These pods usually include jobs."
```

Comment:
`Matches all pods that have an active deadline. These pods have a limited lifespan on a node before being actively terminated by the system` WDYT?